saml/config.go

```diff
@@ -1,105 +1,73 @@
 package saml
 
 import (
-	"crypto/rsa"
-	"crypto/tls"
 	"errors"
 	"fmt"
 	"net/url"
 	"time"
 
 	"github.com/hashicorp/cap/oidc"
-	"github.com/hashicorp/cap/saml/models/core"
 )
 
 var ErrInvalidTLSCert = errors.New("invalid tls certificate")
 
 type ValidUntilFunc func() time.Time
 
-type IDPConfig struct {
-	// SSOServiceURL is the SSO service endpoint where the authentication request
-	// should be sent to. The endpoint must match the location for the given
-	// service binding.
-	SSOServiceURL string
-
-	// TODO: Add more fields for the cert, crypto algorithm, etc.
-	// SLOServiceURL string
-}
-
 type Config struct {
-	// ServiceBinding defines the mechanism that should be used
-	// when exchanging messages between the requester (SP) and responder (IDP).
-	ServiceBinding core.ServiceBinding
-
-	// NameIDFormat controls how the users at the IDP are mapped to
-	// users at the SP during SSO.
-	NameIDFormat core.NameIDFormat
```

Comment:
I made this the default for the AuthnRequest and the plan is to add options to the create AuthnRequest methods later.